.env.example

```diff
@@ -655,6 +655,72 @@ MODULE_ton-main_NODES[]=http://login:password@127.0.0.2:1234/
 MODULE_ton-main_REQUESTER_TIMEOUT=60
 MODULE_ton-main_REQUESTER_THREADS=12
 
+#######################
+## Main TRON Module
+#######################
+
+MODULES[]=tron-main
+MODULE_tron-main_CLASS=TronMainModule
+MODULE_tron-main_NODES[]=http://login:password@127.0.0.1:1234/
```

Comment:
@Har01d please make sure that your nodes are behind proxy with the following rules:

-  `/` is proxied to node's `/jsonrpc` path
- `/wallet/` is proxied to node's `/wallet/` path

Reply:
If it is necessary to use different passes to API, perhaps, it is better to use different environmental variables:
`MODULE_tron-main_JSONRPC[] = http://login:password@127.0.0.1:1234/jsonrpc/`
`MODULE_tron-main_WALLET[] = http://login:password@127.0.0.1:1234/wallet/`
And use in code:
```php
$tron_jsonrpc = envm($this->module, 'JSONRPC');
$info = requester_single($tron_jsonrpc,  params: ['jsonrpc' => '2.0', 'method' => 'eth_blockNumber', 'id' => 0], result_in: 'result', timeout: $this->timeout);
```

Reply:
Sounds reasonable, but @Har01d what do you think if I create additional class atribute in order to not duplicate `$tron_jsonrpc = envm($this->module, 'JSONRPC');` in every invocation of `jsonrpc` endpoint?